File: 100-Days-Series/Day 66/AlphabetInvertedPyramidPattern.js
// Arrow function to print Alphabet Inverted Pyramid Pattern
const getPrintPattern=(rows)=>{
    for(let i=rows;i>=1;--i){
        line="";
        ch=65;
        for(let j=1;j<=i;j++){
            line+=String.fromCharCode(ch++) + " "; 
            // Convert ASCII value to character and add a space
        }
        console.log(line);
    }
}

// Let's Start Here
rows=5;
getPrintPattern(rows);

File: 100-Days-Series/Day 66/AlphabetInvertedPyramidPattern.py

# Function to print Alphabet Inverted Pyramid Pattern
def get_print_pattern(rows):
    for i in range(rows,0,-1):
        ch=65  # ASCII value of 'A'
        for j in range(1,i+1):
            print(chr(ch),end=" ")  # Convert ASCII to character
            ch+=1
        print()


# Let's Start Here
rows=int(input("Enter the number of rows : "))
get_print_pattern(rows)